src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -181,7 +181,8 @@ private void SetGlobalErrorIfTrue(bool arg)
             // module or crashing.
             if (moduleBeingBuiltOpt != null && (methodCompiler._globalHasErrors || moduleBeingBuiltOpt.SourceModule.HasBadAttributes) && !diagnostics.HasAnyErrors() && !hasDeclarationErrors)
             {
-                diagnostics.Add(ErrorCode.ERR_ModuleEmitFailure, NoLocation.Singleton, ((Cci.INamedEntity)moduleBeingBuiltOpt).Name);
+                diagnostics.Add(ErrorCode.ERR_ModuleEmitFailure, NoLocation.Singleton, ((Cci.INamedEntity)moduleBeingBuiltOpt).Name,
+                    methodCompiler._globalHasErrors ? CodeAnalysisResources.UnableToDetermineSpecificCauseOfFailure : CodeAnalysisResources.ModuleHasInvalidAttributes);
```

Comment:
>CodeAnalysisResources.UnableToDetermineSpecificCauseOfFailure : CodeAnalysisResources.ModuleHasInvalidAttributes [](start = 54, length = 112)

This is likely to cause localization problems, I think we are using LocalizableResourceString in situations like this.